### PR TITLE
Fix args

### PR DESCRIPTION
**Pull request by DoriftoShoes**
Actually pass in $@ to setup_args

### CODE REVIEW

**Comment by estee-tew** [comment]
https://stackstorm.slack.com/archives/stackstorm/p1457553712007131

**Comment by m4dcoder** [comment]
+1

File: scripts/st2_bootstrap.sh [hunk 1]
```diff
@@ -33,7 +38,7 @@ setup_args() {
     done
 
   if [[ "$VERSION" != '' ]]; then
-    if [[ ! "$VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]] || [[ ! "$VERSION" =~ ^[0-9]+\.[0-9]+dev$ ]]; then
+    if [[ ! "$VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]] && [[ ! "$VERSION" =~ ^[0-9]+\.[0-9]+dev$ ]]; then
```
Comment (lakshmi-kannan) [inline scripts/st2_bootstrap.sh]: || is the right thing.

File: scripts/st2bootstrap-deb.sh [hunk 2]
```diff
@@ -2,11 +2,16 @@
 
 set -eu
 
-USERNAME='st2admin'
+USERNAME='test'
```
Comment (lakshmi-kannan) [inline scripts/st2bootstrap-deb.sh]: st2admin should be the user.

File: scripts/st2bootstrap-deb.sh [hunk 3]
```diff
@@ -109,59 +189,21 @@ EOT"
...
-    echo "https://docs.stackstorm.com/install/deb.html#setup-chatops"
-    echo "########################################################"
-  fi
-}
-
 verify_st2() {
   st2 --version
   st2 -h
 
-  st2 auth $USERNAME -p $PASSWORD
```
Comment (lakshmi-kannan) [inline scripts/st2bootstrap-deb.sh]: We need these variables for chatops config. Some git goof up for sure.

File: scripts/st2bootstrap-deb.sh [hunk 4]
```diff
@@ -58,7 +149,7 @@ configure_st2_authentication() {
   sudo apt-get install -y apache2-utils crudini
 
   # Create a user record in a password file.
-  sudo echo $PASSWORD | sudo htpasswd -i /etc/st2/htpasswd $USERNAME
+  sudo echo "${PASSWORD}" | sudo htpasswd -i /etc/st2/htpasswd test
```
Comment (lakshmi-kannan) [inline scripts/st2bootstrap-deb.sh]: s/test/${USERNAME}

File: scripts/st2bootstrap-deb.sh [hunk 5]
```diff
@@ -15,15 +20,101 @@ fail() {
...
+    fi
+
+    if [[ "$VERSION" =~ ^[0-9]+\.[0-9]+dev$ ]]; then
+     echo "You're requesting a dev version! Switching to unstable!"
+     RELEASE='unstable'
+    fi
+  fi
+
+  echo "########################################################"
+  echo "          Installing st2 $RELEASE $VERSION              "
```
Comment (lakshmi-kannan) [inline scripts/st2bootstrap-deb.sh]: s/st2/StackStorm as per dzimine

File: scripts/st2bootstrap-el6.sh [hunk 6]
```diff
@@ -40,7 +40,7 @@ setup_args() {
     done
 
   if [[ "$VERSION" != '' ]]; then
-    if [[ ! "$VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]] || [[ ! "$VERSION" =~ ^[0-9]+\.[0-9]+dev$ ]]; then
+    if [[ ! "$VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]] && [[ ! "$VERSION" =~ ^[0-9]+\.[0-9]+dev$ ]]; then
       echo "$VERSION does not match supported formats x.y.z or x.ydev"
```
Comment (lakshmi-kannan) [inline scripts/st2bootstrap-el6.sh]: Dude, EL scripts use apt-cache to get package versions. Luckily no one is using --version.

File: scripts/st2bootstrap-el7.sh [hunk 7]
```diff
@@ -67,21 +67,21 @@ setup_args() {
 get_full_pkg_versions() {
   if [ "$VERSION" != '' ];
   then
-    local ST2_VER=$(apt-cache show st2 | grep Version | awk '{print $2}' | grep $VERSION | sort --version-sort | tail -n 1)
+    local ST2_VER=$(yum info st2 | grep Version | awk '{print $3}' | grep 1.3.2 | sort --version-sort | tail -n 1)
```
Comment (lakshmi-kannan) [inline scripts/st2bootstrap-el7.sh]: grep ${VERSION} bro. WHAZ happenin!

File: scripts/st2bootstrap-el6.sh [hunk 8]
```diff
@@ -67,21 +67,21 @@ setup_args() {
 get_full_pkg_versions() {
   if [ "$VERSION" != '' ];
   then
-    local ST2_VER=$(apt-cache show st2 | grep Version | awk '{print $2}' | grep $VERSION | sort --version-sort | tail -n 1)
+    local ST2_VER=$(yum info st2 | grep Version | awk '{print $3}' | grep 1.3.2 | sort --version-sort | tail -n 1)
```
Comment (lakshmi-kannan) [inline scripts/st2bootstrap-el6.sh]: grep ${VERSION}

**Pull request merged**